contracts/examples/Collateralized.sol

```diff
@@ -165,11 +176,15 @@ contract Collateralized is TermsContract {
 
         // seize collateral and transfer to beneficiary.
         require(
-            ERC20(collateral.token).transfer(beneficiary, collateral.amount)
+            ERC20(collateral.token).transferFrom(
```

Comment:
Wouldn't this `transferFrom` require `approve` call from `Collateralized` contract? 
Maybe I am missing something. I haven't take a deep look at the PR.

Reply:
I think so. Would love for @nadavhollander to weigh in with his l33t solidity skillz as to whether we want to be using `transfer` or `transferFrom`.

As I understand it, if we don't use `transferFrom`, the value being transferred will be from the `msg.sender` whereas the value that should be transferred should be from the contract itself which is holding the collateral in escrow.

But please correct me if I'm mistaken.

Reply:
Well, `msg.sender` in `transfer` method will be the caller (contract or account), not TX initiator. In this case, `msg.sender` will be the `Collateralized` contract, which exactly we want. So, it should be `transfer`, not `transferFrom`.

Reply:
That makes sense to me. Will let @nadavhollander also weigh in to ensure we're all on the same page.

Reply:
Good catch @jdkanani, indeed `transfer` would be the correct method to use here.  For more context on the dynamics of how `msg.sender` is resolved in different calling scopes, see [here](https://ethereum.stackexchange.com/questions/1891/whats-the-difference-between-msg-sender-and-tx-origin)